scripts/run_migrations.sh

```
@@ -0,0 +1,50 @@
+
+echo "Running Alembic migrations..."
+
+# Configuration
+MAX_MIGRATION_RETRIES=${MAX_MIGRATION_RETRIES:-5}
+INITIAL_DELAY=${MIGRATION_INITIAL_DELAY:-2}
+MAX_DELAY=${MIGRATION_MAX_DELAY:-30}
+
+attempt=1
+delay=$INITIAL_DELAY
+
+while [ $attempt -le $MAX_MIGRATION_RETRIES ]; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)